javaagent/sdk-config.yaml

```diff
@@ -0,0 +1,35 @@
+file_format: "0.1"
+
+resource:
+  attributes:
+    service.name: agent-example-app
+
+logger_provider:
+  processors:
+    - batch:
+        exporter:
+          otlp:
+            endpoint: http://collector:4317
```

Comment:
```suggestion
            endpoint: http://collector:4318
```

Reply:
Need to update the protocol to http/protobuf along with this. I've gone ahead and done that.